diff --git a/node_modules/jest-config/build/ValidConfig.js b/node_modules/jest-config/build/ValidConfig.js
index 0c1250c..823448a 100644
--- a/node_modules/jest-config/build/ValidConfig.js
+++ b/node_modules/jest-config/build/ValidConfig.js
@@ -324,6 +324,7 @@ const initialProjectOptions = {
     userAgent: 'Agent/007'
   },
   testLocationInResults: false,
+  testTimeout: 5000,
   testMatch: ['**/__tests__/**/*.[jt]s?(x)', '**/?(*.)+(spec|test).[jt]s?(x)'],
   testPathIgnorePatterns: [NODE_MODULES_REGEXP],
   testRegex: (0, _jestValidate().multipleValidOptions)(
